presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java

```diff
@@ -195,63 +195,55 @@ public void testCreateTable()
     @Test
     public void testCreateTableAsSelect()
     {
-        assertUpdate("CREATE TABLE IF NOT EXISTS test_ctas AS SELECT name, regionkey FROM nation", "SELECT count(*) FROM nation");
-        assertTableColumnNames("test_ctas", "name", "regionkey");
-        assertUpdate("DROP TABLE test_ctas");
+        String tableName = "test_ctas" + randomTableSuffix();
+        assertUpdate("CREATE TABLE IF NOT EXISTS " + tableName + " AS SELECT name, regionkey FROM nation", "SELECT count(*) FROM nation");
```

Comment:
consider using `String.format` instead

Reply:
when fmt-ing SQLs in tests, using + is often more readable than %s (especially when you substitude very few things in long text)

Reply:
Also, I refactored this code with some search-and-replace methods. Using `format` would be much harder to make it automatically.